docs/source/notebooks/mmm/mmm_lift_test.ipynb

```diff
@@ -0,0 +1,1776 @@
ETPAnMPMun2DTp0ytn/1yNx0fIKGkkpKKNqfc6/VQ7aoBzd0Gf1qcw3mjKltFKR2uDGIsN5bRraTY0EWJBzTLcds+MH9d2gfy5SkALpt8y/4aKmsA2G7TW2KMFFrsePnov0VSIBTXLuGKRcMbChKO/GLldZ8aPA9DoaS2bIQ16SH38wmKX1sZYkc4urX0tR0dPvtEc/yKcvjhYLTZ9sz6XVkY9r0u0MAoEQuyaLUvr1wJwfHS/wZXMnV29al7XxrbymsI4GS2HKh8TTXJBCxVu9y3WT/rMhiYMziWgXnN11bjqsFnsea1rOtqmyO3KyGmeO/NHQLQwmhW/SxVEM0qa0AwndJUC2vt8tu7NSnd2ZeQ0fzzxSwBuWCpaGM9HMSYypuUU/3fgvwC4fc17RMtSlrXZkPoDg9tJy6mCraMoCvfu/joAFzVdWrB1is3ERMaZtzE2VlWes6vR04ZVspGSk4xE+40uR0efDlpd+hlypYL2XPfnEG+gT2IULYwCgRC7ZsvS+jUAJdvGqCgKO3vUvK5N7eWX16WRi1spnxzOtjCa1dUFE5ldc3HBjejh9MVpYdRonSK3a9/Ac4SSAXzOOlY3XVLUegQzw2514M1e/JRTSP2RWYbTa2gtub3BU/ktqETY0fMYY7Eh/K56Lpt/ndHlmJZl9euwSBaGIr36Z26+eeLU7xiM9OB3NXD9ktcWZI1SpKN2BV6Hn3g6yvECnvc93fUIL/Q9id3q5O0bP1qwdYqNd4bOroyc0UWexgp0dlktNv1czEy5XVobY4toYywa2nPdnwdnlwinFwiE2DVrltapzq7OscMlGVLfFTjKcLQPu9XJ2ubLjC6nYGh5FwOR2buV8okWamtmsUt7rkajA7N+TWsDAOqqiit2aV/gk08Kn+l6FIAt868vmxaQckQLqR+Nlkdul6Iosw6n12jzaW2M5rm4KSaPZIPpr13y2qI6Q0sNt93DgpplQGHcXYqicN/+7wLwmtV/g8PmyvsapYpFsrCmRT1X2tP/TEHWSGYSfP/5zwBw+0Xv1j8XyoEaVx1wYWdXID6MrGSwSJayjde4ENokPlOJXWU0MKFU0J7r/hw6U/RJjMLZJRAIsWu2tHoX4HH4SMspTgeOGl3OrNmZncK4tvkynDa3wdUUjmYTOLsURZlwdjWYcxIjqK1lDqsLBUVvI5gpI1HVnVNX5JPTVr2NsRNQn+tnsnldWxfcVNRaBLNDy+0KxMtD7OoLdRFKjGG3OFhcu3pWj23zqm2MA+EzZOR0IcozLcHEmP6evVG0MF4QvZUx2zKbT/b0P82xkb04rW5eufKOvB+/1FmfbWUsVG7X/QfvpjfUSb27mTet//uCrGEUPqcmdk3v7NLC6evczRW7WaWd1/SbJKQ+lUkyks1RE5ldxUNrY8wlhkU7NxbOLoFAiF2zRpIkltaprYyFtLQXip3ZvK5NZZzXBbnlUOWLvlAX4/FR7BYHS+ovMqyOCyFJ0qSJjLP7ch3LtqIV3dn1IkfM6cARekOd2K1OLml/aVFrEcyO2qzYNRotjzbGI1n35uK61ditjlk9tsHTit3qJC2nDB+mUWweO/kAKTnJkrqL9HgAwflZ0VC4iYy/3P89AG5Y9joxJGAKtImM+waeIyNn8nrssdgQ9+7+GgDvuORfym5ymj6N8QJtjMORyg2n12j1mmsi41CkF1mRcVhdFeu2M4KJNsa5vQ6SmYQuHgtnl0AgxK45oYXUl1puVyqTZE/f0wBsLOO8LphozYumQoSTQUNq0C6Cl9RdhMPqNKSGmTJXJ5yWH1Nf7Myu7G6V6ojJ8HS2hfHi1itnNQ1PUHz0NsYyyew6MjT3VmWLZKE1u4s7edhCJfBotoVRBNPPjMkh9Yqi5O24XYFjPHvmD0hI3H7Ru/N23HJiad0a3DYPkWSQzrHDeT32j3Z+iWgqzIqGDVy/9HV5PbYZmAiov4CzK6penFdiXpeG5uwySxtj/6RJjGK6dfHQ2hgHIz1zEtcHwmdQUHDZqvShQAJBJSPErjmg7UIfHymtiYyHhnYST0fxu+pZUmdep1E+cNs9un3eqFbGQ9kWxtlOaDOCuTrhNMGi2Lt+DVWt2C0O1RET7ubprt8DcLloYTQ92mtFm+RZ6hwZ3gPMPpxeo22KYQvlzunAUQ4Pv4BVsnHdktuMLqckWFy7GrvFQSgxllfnxy/3fx9Q27/n1SzJ23HLCavFxkXN6oTEvQP5y+06NryX3x/9KQDv2/L5spyAOVOxa7iCJzFqmC2za0DP65pncCWVRUNVK1bJRlpO6bm4s2FyC6MQKQUCIXbNiWVZZ9fJ0QPIimxwNTNHm8J4cetVZXlS9WI0AWcwh773XNAcH6sazZvXpTHXiYxaG2N9kdsYrRarvvu1u+cZPRtty4Ibi1qHYPbobYxl4OzKyBmOjewFchC7si7FngqayPjosV8AcOn860R7zAyxWx0srlMz4TTXcK6MxYb4w4n7AHjdmr/LyzHLlXVaK2OecrsUReE7z30SBYVrF99WthOEJzK7pg+oHxJtjLqzazQ2SDwdNbiaiXZKTYQTFAerxapfv8xlIqMudokWRoEAEGLXnJhfsxSH1UUsHSmpkfFaXtfG9vLO69Jo9qgCzoABWThpOaW3ua4sAbFLn145S2eX1sZY7MwumMjt+umubwOq2NBQ1VL0OgSzo65KFbvKwdnVNX6UeDqK2+ZhXs3SOR2j1Weu1pVCk5Ez/PHELwERTD9bNEFV20jJld8c+jGpTIKVDRezJutcEkzN2mZV7Nrb/2xe2kgfP/Ub9g88h8vm5l2bP5Hz8cyKltk1Hp9e7BrW2xgr19nldfjxOHwADMxB5Mg3A5PaGAXFpUWfyDh7F29v1iWuiacCQaUjxK45YLXYWFy7Ciid3K5QIsDRbLDtprbyzuvSmHB2FT+k/uToQZKZOF6Hn3bfoqKvP1ua5pDZFU2F9d1HbcJeMdFyuw70bwfg8gUvK3oNgtmjvVbGys
```

Comment:
I would be slightly more pedagogical here. I suggest:

## About Lift Tests

In a lift study, one temporarily changes the budget of a channel for a fixed period of time, and then uses some method (for example [CausalPy](https://causalpy.readthedocs.io/en/latest/)) to make inference about the change in sales directly caused by the adjustment.

A lift test is characterized by:

- `channel`: the channel that was tested
- `x`: pre-test channel spend
- `delta_x`: change made to `x` 
- `delta_y`: inferred change in sales due to `delta_x`
- `sigma`: standard deviation of `delta_y`

An experiment characterized in this way can be viewed as two points on the saturation curve for the channel. Accordingly, lift test calibration is implemented by adding a term to the model likelihood, that makes the channel saturation curve (the contribution as a function of spend) align with the test observation.